providers/asrockrack/helpers.go

```diff
@@ -83,8 +83,8 @@ func (a *ASRockRack) setFlashMode() error {
 func multipartSize(fieldname, filename string) int64 {
 	body := &bytes.Buffer{}
 	form := multipart.NewWriter(body)
-	form.CreateFormFile(fieldname, filename)
-	form.Close()
+	_, _ = form.CreateFormFile(fieldname, filename)
+	_ = form.Close()
```

Comment:
This commit looks like something to (ideally) fold back into 968ea38dc70cdb76f608b1379b532c20978e949a via a fixup/rebase.

Reply:
squashed into 7f3f530